content/reference/cpp/cpp-generated.md

```diff
@@ -1048,6 +1084,10 @@ In addition, it will generate these methods:
 
 ## Enumerations {#enum}
 
+**Note:** As of edition 2024
+[string_view](/reference/cpp/string-view#enum-name) APIs
+may be generated instead
```

Comment:
Did this one escape our last pass to change `may` to `will`, or is this one unique?

Reply:
Yeah. I'll patch in the update.

Reply:
This is now updated.